reporters/reporter-default/src/main/java/org/radargun/reporting/csv/CsvReporter.java

```diff
@@ -66,7 +83,7 @@ private void reportTest(Report report, Report.Test test) {
          fileWriter = prepareOutputFile(report, test.name, "");
          int it = 0;
          Set<String> columns = new TreeSet<String>();
-         ArrayList<Map<String, String>> rows = new ArrayList();
+         ArrayList<Map<String, String>> rows = new ArrayList<Map<String, String>>();
```

Comment:
This should be generating compile time warning and, more importantly, annoys my eclipse. I am pretty sure this should be done...come on...